### PR TITLE
cleaned up a few projects

### PR DESCRIPTION
**Pull request by albeethekid**
Added a few test cases as well as wiring up the metadata for several projects

### CODE REVIEW

File: src/cadence/scripts/getNFTs.cdc [hunk 1]
```diff
@@ -805,22 +831,32 @@ pub fun getRareRooms(owner: PublicAccount, id: UInt64): NFTData? {
 
     let col = owner.getCapability(RareRooms_NFT.CollectionPublicPath)
         .borrow<&{RareRooms_NFT.RareRooms_NFTCollectionPublic}>()
-    if col == nil { return nil }
+    //if col == nil { return nil }
 
     let nft = col!.borrowRareRooms_NFT(id: id)
-    if nft == nil { return nil }
+    //if nft == nil { return nil }
```
Comment (psiemens) [inline src/cadence/scripts/getNFTs.cdc:837]: You'll want to keep these lines

**Review by psiemens — COMMENTED** [review]
Nice!

File: src/cadence/scripts/getNFTIDs.cdc [hunk 2]
```diff
@@ -167,13 +167,13 @@ pub fun main(ownerAddress: Address): {String: [UInt64]} {
     }
 
     if let col = owner.getCapability(KOTD.CollectionPublicPath)
```
Comment (niveda-krish) [inline src/cadence/scripts/getNFTIDs.cdc:169]: Can we remove lines 169-176? Just realized these are duplicate lines (KOTD and Crave are already accounted for here: Line 60 and 84

**Comment by niveda-krish** [comment]
Thanks for doing this, looks great

File: src/cadence/scripts/getNFTIDs.cdc (hunk 2 above)
Comment (niveda-krish) [inline src/cadence/scripts/getNFTIDs.cdc:169]: Can we delete lines 169-177, just realized these are already accounted for on Lines 60 and 84

**Comment by niveda-krish** [comment]
Could you also update the testnet scripts if they apply?

File: src/cadence/scripts/getNFTs.cdc [hunk 3]
```diff
@@ -361,67 +366,74 @@ pub fun getCrave(owner: PublicAccount, id: UInt64): NFTData? {
...
+        )
+
+        let metadata = nft!.getMetadata()
+        log(metadata["ipfs"]) //why don't I see this in the output?
+
+        return NFTData(
+            contract: contract,
+            id: nft!.id,
+            uuid: nft!.uuid,
+            title: nil,
```
[the remark this answers is not in the document]
Comment (albeethekid) [reply src/cadence/scripts/getNFTs.cdc:390]: The only NFTs they've minted thus far are vouchers, without a title.

**Review by niveda-krish — APPROVED** [review]

**Pull request merged**